docs/quick_start/north.rst

```diff
@@ -19,9 +19,17 @@ Adding Data Destinations
 To add a data destination, click on “Create North Instance+” in the upper right of the North Services screen.  Fledge will display a series of 3 screens to add the data destination:
 
 1. The first screen will ask you to select the plugin for the data destination from the list of installed plugins.  If you do not see the plugin you need, refer to the Installing Fledge section of this manual.  In addition, this screen allows you to specify a display name for the data destination. In addition, you can specify how frequently data will be forwarded to the destination in days, hours, minutes and seconds.  Enter the number of days in the interval in the left box and the number of hours, minutes and seconds in format HH:MM:SS in the right box.
-2. The second screen allows you to configure the plugin and the data assets it will send.  See the section below for specifics of configuring a PI, EDS or OCS destination.
+2. The second screen allows you to configure the plugin and the data assets it will send.  See the section below for specifics of configuring a PI, EDS, OCS or ADH destination.
+   .. note::
```

Comment:
note directive will display as is (i.e as a plain text)

See the patch here - https://github.com/fledge-iot/fledge/compare/FOGL-9029.doc-patch?expand=1 and also fixes some whitespaces in the doc